website/_dogfooding/_docs tests/index.md

```diff
@@ -1,5 +1,6 @@
 ---
 slug: /
+tags: [test-tag1, testTag2, testTag3]
```

Comment:
We will support YAML array? I mean syntax something like this (similar to frontmatter field `keywords`):

```yml
 tags:
  - test-tag1
  - testTag2
```

Reply:
Keeping the docs tag frontmatter consistent with blog tags. Both should lead to the same, but I'll add more variation in _dogfooding to ensure that